package.json

```diff
@@ -85,9 +91,21 @@
       ]
     },
     "angular-mock": "global:angular.mock",
-    "diff-match-patch": "diff_match_patch",
-    "dom-text-mapper": "DomTextMapper",
-    "dom-text-matcher": "DomTextMatcher",
+    "dom-anchor-fragment": "domAnchorFragment",
```

Comment:
By that I mean, I made sure that they don't, but it's only through all this nasty stuff.